incubator/prisma/values.yaml

```diff
@@ -0,0 +1,97 @@
+# ------------------------------------------------------------------------------
+# Prisma:
+# ------------------------------------------------------------------------------
+
+# The deployment installs one pod by default
+replicas: 1
+
+image:
+  # Prisma image repository
+  repository: prismagraphql/prisma
+  # Prisma image version
+  tag: 1.8
+  # Specify an imagePullPolicy
+  # ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  pullPolicy: IfNotPresent
+
+## Databse configuration for Prisma. In most scenarios, these values
+## don't need to be changed unless you want to use a differnt database
+## than the one in the 'requirementes.yaml' for this chart. If that's the
+## case, please set 'postgresql.enabled' to false at the bottom of this file.
+database:
+  # The current supported connectors are [mysql, postgres]
+  connector: postgres
+  # These are the credentials needed to authenticate with the database.
+  user: prisma
+  password: prisma
+  # If 'postgresql.enabled' is false, you will need to provide the
+  # following values so that Prisma can use them as the database endpoint.
+  # host: ""
+  # port: ""
+
+## Optional secret to authenticate requests done through the Prisma CLI
+auth:
+  enabled: false
+  # secret: ""
+
+## Expose the prisma service to be accessed from outside the cluster (LoadBalancer service).
+## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: ClusterIP
+  port: 4466
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts: []
+    # - prisma.local
+  tls: []
+  #  - secretName: prisma-tls
+  #    hosts:
+  #      - prisma.local
+
+resources: {}
+#  limits:
+#    cpu: 100m
+#    memory: 128Mi
+#  requests:
+#    cpu: 100m
+#    memory: 128Mi
+
+## Node labels for pod assignment
+## ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+## Tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+## Affinity for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+# ------------------------------------------------------------------------------
+# PostgreSQL:
+# ------------------------------------------------------------------------------
+
+postgresql:
+  ## If true, install the PostgreSQL chart alongside Prisma
+  ## ref: https://github.com/kubernetes/charts/tree/master/stable/postgresql
+  enabled: true
```

Comment:
I think the default should be `false`. But good idea to install the chart automatically!

Reply:
According to the technical requirements of the contribution guidelines, the pods should enter a running state with a `helm install` using the default values. If a database connector is not available, Prisma will be in a crash loop state. That's why we need a connector deployed by default.

Reply:
Well, the thing is that installing the postgres/mysql DB charts should be optional. It's required if you don't have any installed yet, but if you already have a running DB you don't need the extra chart.
And yeah, prisma will eventually crash if it can't find/connect to the DB. But that's fine 😉

Reply:
I agree 👍 
However, do you think it will pass CI going into CrashLoop?

Reply:
perhaps a simple non-persistent db deployment for testing and CI purposes baked into this chart that can easily be turned off to point to a real deployment.

Reply:
I mean, the PostgreSQL dependency is a real deployment. But anyways, I'll set `persistence.enabled` to `false` to make it simpler.